libs/cdk-utils/src/lib/constructs/ssm-parameter-group.test.ts

```diff
@@ -0,0 +1,100 @@
+import { App, Stack, Stage } from 'aws-cdk-lib';
+import { Template } from 'aws-cdk-lib/assertions';
+import { Code, Function, Runtime } from 'aws-cdk-lib/aws-lambda';
+import { StringParameter } from 'aws-cdk-lib/aws-ssm';
+import { Construct } from 'constructs';
+import { beforeEach, describe, expect, test } from 'vitest';
+
+import { SsmParameterGroup } from './ssm-parameter-group';
+
+class TestParameters extends SsmParameterGroup<'PARAM_ONE' | 'PARAM_TWO'> {
+    public readonly parameters;
+
+    constructor(scope: Construct, id: string) {
+        super(scope, id);
+
+        const PARAM_ONE = new StringParameter(this, 'Param1', {
+            parameterName: '/test/param1',
+            stringValue: 'value1',
+        });
+
+        const PARAM_TWO = new StringParameter(this, 'Param2', {
+            parameterName: '/test/param2',
+            stringValue: 'value2',
+        });
+
+        this.parameters = {
+            PARAM_ONE,
+            PARAM_TWO,
+        } as const;
+    }
+}
+
+describe('SsmParameterGroup', () => {
+    let stack: Stack;
+    let template: Template;
+    let testParameters: TestParameters;
+    let lambda: Function;
+    let lambda2: Function;
+
+    beforeEach(() => {
+        const app = new App();
+        const stage = new Stage(app, 'TestStage');
+        stack = new Stack(stage, 'TestStack');
+        testParameters = new TestParameters(stack, 'MyTestParameters');
+        lambda = new Function(stack, 'MyTestFunction', {
+            runtime: Runtime.NODEJS_20_X,
```

Comment:
In applications/core/bin/main.ts we use Node 22, is there any reason to not have our tests use that too?